### DISCUSSION TITLE
[Unstable] Graveyard sub-biome bad for performance

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14139] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The WastesGraveyard sub-biome is not very performance friendly
It has:

- 3 to 4 wrecks
- 1 very large ruin
- 5 Abyss islands
- 2 Caves
- Most likely a beacon
- Dozens of hours of accumulated late-game resources

Suggest trying to cut down to 3 wrecks, 0 ruins, 1 cave 1 abyss island (for mission support) 

### Version

Unstable

**Comment by JeroonBosch** [comment]
We decided it wasn't worth keeping this particular variant (hard to make it interesting / an actual graveyard when the performance does not allow for it). Closing
